### ISSUE TITLE
error: package directory '...' does not exist

### ISSUE THREAD

**Issue by lucaswiman** [nvbn/py-backwards-packager#1]
This is a simplified example from an attempt to use py-backwards in a project I'm working on. 

The cause seems to be that `py-backwards-manager` is wrapping the sdist source into a directory called `tmph62ul56t`, which the rest of the generated package is aware of. 
## The failure

```
$ pip install dist/example-0.1.tar.gz 
Processing ./dist/example-0.1.tar.gz
    Complete output from command python setup.py egg_info:
    /Users/lucaswiman/.pyenv/versions/2.7.11/lib/python2.7/distutils/dist.py:267: UserWarning: Unknown distribution option: 'py_backwards_targets'
      warnings.warn(msg)
    running egg_info
    creating pip-egg-info/example.egg-info
    writing pip-egg-info/example.egg-info/PKG-INFO
    writing top-level names to pip-egg-info/example.egg-info/top_level.txt
    writing dependency_links to pip-egg-info/example.egg-info/dependency_links.txt
    writing manifest file 'pip-egg-info/example.egg-info/SOURCES.txt'
    warning: manifest_maker: standard file '-c' not found
    
    error: package directory 'example' does not exist
    
    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /private/var/folders/6l/mfzbddtx5f58fr6zkq79pjvc0000gn/T/pip-OTjNgD-build/
```

## Full Details
```
$ find .
.
./example
./example/__init__.py
./setup.py
$ cat example/__init__.py 
x = 1

def doit():
    print(f'{x}')
$ cat ./setup.py 
try:
    from py_backwards_packager import setup
except ImportError:
    from setuptools import setup
setup(name='example',
      version='0.1',
      description='example',
      author='x',
      packages=['example'],
      zip_safe=True,
      py_backwards_targets=['2.7', '3.5'],      
)
$ python3 ./setup.py sdist
running sdist
running egg_info
creating tmph62ul56t/example.egg-info
writing ./tmph62ul56t/example.egg-info/PKG-INFO
writing dependency_links to ./tmph62ul56t/example.egg-info/dependency_links.txt
writing top-level names to ./tmph62ul56t/example.egg-info/top_level.txt
writing manifest file './tmph62ul56t/example.egg-info/SOURCES.txt'
reading manifest file './tmph62ul56t/example.egg-info/SOURCES.txt'
writing manifest file './tmph62ul56t/example.egg-info/SOURCES.txt'
warning: sdist: standard file not found: should have one of README, README.rst, README.txt

running check
warning: check: missing required meta-data: url

warning: check: missing meta-data: if 'author' supplied, 'author_email' must be supplied too

creating example-0.1
creating example-0.1/tmph62ul56t
creating example-0.1/tmph62ul56t/example
creating example-0.1/tmph62ul56t/example.egg-info
creating example-0.1/tmph62ul56t/example/_compiled_2_7
creating example-0.1/tmph62ul56t/example/_compiled_2_7/example
creating example-0.1/tmph62ul56t/example/_compiled_3_5
creating example-0.1/tmph62ul56t/example/_compiled_3_5/example
copying files to example-0.1...
copying setup.py -> example-0.1
copying ./tmph62ul56t/example/__init__.py -> example-0.1/./tmph62ul56t/example
copying ./tmph62ul56t/example.egg-info/PKG-INFO -> example-0.1/./tmph62ul56t/example.egg-info
copying ./tmph62ul56t/example.egg-info/SOURCES.txt -> example-0.1/./tmph62ul56t/example.egg-info
copying ./tmph62ul56t/example.egg-info/dependency_links.txt -> example-0.1/./tmph62ul56t/example.egg-info
copying ./tmph62ul56t/example.egg-info/top_level.txt -> example-0.1/./tmph62ul56t/example.egg-info
copying ./tmph62ul56t/example.egg-info/zip-safe -> example-0.1/./tmph62ul56t/example.egg-info
copying ./tmph62ul56t/example/_compiled_2_7/example/__init__.py -> example-0.1/./tmph62ul56t/example/_compiled_2_7/example
copying ./tmph62ul56t/example/_compiled_3_5/example/__init__.py -> example-0.1/./tmph62ul56t/example/_compiled_3_5/example
Writing example-0.1/setup.cfg
creating dist
Creating tar archive
removing 'example-0.1' (and everything under it)
$ pip install dist/example-0.1.tar.gz 
Processing ./dist/example-0.1.tar.gz
    Complete output from command python setup.py egg_info:
    /Users/lucaswiman/.pyenv/versions/2.7.11/lib/python2.7/distutils/dist.py:267: UserWarning: Unknown distribution option: 'py_backwards_targets'
      warnings.warn(msg)
    running egg_info
    creating pip-egg-info/example.egg-info
    writing pip-egg-info/example.egg-info/PKG-INFO
    writing top-level names to pip-egg-info/example.egg-info/top_level.txt
    writing dependency_links to pip-egg-info/example.egg-info/dependency_links.txt
    writing manifest file 'pip-egg-info/example.egg-info/SOURCES.txt'
    warning: manifest_maker: standard file '-c' not found
    
    error: package directory 'example' does not exist
    
    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /private/var/folders/6l/mfzbddtx5f58fr6zkq79pjvc0000gn/T/pip-OTjNgD-build/
$ cd dist/
$ tar zxvf ./example-0.1.tar.gz 
example-0.1/
example-0.1/PKG-INFO
example-0.1/setup.cfg
example-0.1/setup.py
example-0.1/tmph62ul56t/
example-0.1/tmph62ul56t/example/
example-0.1/tmph62ul56t/example/__init__.py
example-0.1/tmph62ul56t/example/_compiled_2_7/
example-0.1/tmph62ul56t/example/_compiled_2_7/example/
example-0.1/tmph62ul56t/example/_compiled_2_7/example/__init__.py
example-0.1/tmph62ul56t/example/_compiled_3_5/
example-0.1/tmph62ul56t/example/_compiled_3_5/example/
example-0.1/tmph62ul56t/example/_compiled_3_5/example/__init__.py
example-0.1/tmph62ul56t/example.egg-info/
example-0.1/tmph62ul56t/example.egg-info/dependency_links.txt
example-0.1/tmph62ul56t/example.egg-info/PKG-INFO
example-0.1/tmph62ul56t/example.egg-info/SOURCES.txt
example-0.1/tmph62ul56t/example.egg-info/top_level.txt
example-0.1/tmph62ul56t/example.egg-info/zip-safe
```

**Comment by lucaswiman** [comment]
Thanks for your work so far on `py-backwards`! This project looks incredibly useful, and has the potential to be a _big_ step forwards for python 3 adoption.

**Comment by nvbn** [comment]
I've changed it a bit, now packager compiles code only on `bdist` and `bdist_wheel`.

But there was a similar problem with `bdist` and `bdist_wheel`, fixed in 0.2
